extension/json/test/example.test

```diff
@@ -3,42 +3,38 @@
 --
 
 -CASE ExtractNotExistField
--SKIP
 -STATEMENT LOAD EXTENSION "${KUZU_ROOT_DIRECTORY}/extension/json/build/libjson.kuzu_extension";
 ---- ok
 -STATEMENT create node table person (id INT64, description json, primary key(id));
 ---- ok
 -STATEMENT CREATE (p:person {id: 5, description: 6})
 ---- ok
--STATEMENT CREATE (p:person {id: 8, description: {"family": "anatidae", "species": [ "duck", "goose", "swan", null]})
+-STATEMENT CREATE (p:person {id: 8, description: to_json({"family": "anatidae", "species": [ "duck", "goose", "swan", null]})})
 ---- ok
-#TODO(Maxwell): fix this
 -STATEMENT MATCH (p:person) return json_extract(p.description, '$.family');
 ---- 2
 
-anatidae
+"anatidae"
```

Comment:
Why do we have addition quotes around the string?

Reply:
JSON strings must have double quotes around them.